content/blog/2020-03-16-teaching-online-on-short-notice/index.Rmarkdown

```diff
@@ -0,0 +1,215 @@
+    Equally,
+    when you're teaching online for the first time,
+    get someone else to monitor the chat for questions,
+    figure out who to call on next,
+    and so on.
+    If none of your colleagues can do this (remember, they're probably scrambling too),
+    ask one of your more advanced learners to help out.
+
+1.  *Do a dry run (with your copilot).*
+    Fifteen minutes with a small group before you do a real (larger) class
+    can save you from saying, "Nobody told me there was a firewall."
+
+1.  *Keep your notes beside you.*
+    For whatever reason,
+    people seem less distracted when they look to the side at a printed copy
+    than when they look up at a second monitor.
+    There's also less risk of your audience getting to see the inside of your nose…
+
+1.  *Add speaker's notes to your slides.*
+    These can be point-form or long-form depending on how much time you have,
+    but they will make your content easier to find,
+    easier to review,
+    and more accessible to people who have trouble seeing or hearing.
+
+1.  *Use accessibility tools to help everyone.*
+    Speaker's notes are just one example of this:
+    enlarging your cursor,
+    using a [key caster][keycastr] to display control keys as you type,
+    switching to high-contrast mode all help compensate for shaky video
+    or low-quality screens.
+
+1.  *Teach from your smallest screen.*
+    If your slides or your code are on the fancy meter-wide monitor you use for your daily work,
+    they're not going to look good when they're squashed down
+    to fit in the 800-pixel window your learners have on their tablets.
+    Using a smaller video window also reduces bandwidth a little,
+    which may in turn help learners in lower-quality environments.
+
+1.  *Your voice matters more than your face.*
+    You can switch off video if you have to,
+    but if you lose sound,
+    your class is over.
+    Eight dollars will get you a Yeti or Samson Meteor microphone;
+    another thirty will get you a stand
+    (because if it's on your desk, all your audience will hear is your typing).
+    You can also use a headset with a microphone,
+    but this can become uncomfortable after a couple of hours,
+    particularly if you wear glasses.
+
+1.  *Turn off notifications (theirs too).*
+    It's rude to check your phone or read email during an in-person meeting,
+    so don't do it when you're teaching
+    (not even while your learners are working on an exercise).
+    You probably can't persuade your learners to be similarly respectful,
+    but the subset you're talking to at any particular time might.
+
+1.  *Share work in real time.*
+    Have learners share their work in a Google Doc or [HackMD][hackmd] during the lesson.
+    (This is no more or no less intimidating than asking a music student to play something for the class.)
+    When you do this,
+    paste in a list of people's names so that they know where to type.
```

Comment:
Worth suggesting to prepopulate the doc with learners' names (if you know them ahead of time)?

Reply:
Do you mean add the names at the top of the doc? I usually keep a text file with the names in a separate window so that I've got the ones I'm currently calling on.